frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -45,8 +45,6 @@ interface MetricProps extends TaipyBaseProps, TaipyHoverProps {
     testId?: string;
     layout?: string;
     defaultLayout?: string;
-    style?: string;
```

Comment:
Should this be removed from factory.py and viselements.json ?

Reply:
We don't have it in factory.py and viselements.json at the moment.